### PR TITLE
OMERO 5.4.1 release

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/sj0qtpOa/41-open-pr-against-wwwopenmicroscopyorg and https://github.com/openmicroscopy/openmicroscopy/pull/5578/files

### CODE REVIEW

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/2017/11/23/omero-5-4-1.html and https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/downloads/

File: _posts/2017-11-23-omero-5-4-1.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Release of OMERO 5.4.1
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.4.1.
+---
+Today we are releasing OMERO 5.4.1, a bug-fix release.
+
+Improvements include:
+
+* labelled zoom slider bars in the UI to differentiate from horizontal
```
Comment (mtbc) [inline _posts/2017-11-23-omero-5-4-1.md]: FYI, in US English "labelled" *is acceptable* but unusual in comparison with "labeled".

File: _posts/2017-11-23-omero-5-4-1.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+* CLI fixes:
+  * improved help output for graphs commands to make it clearer that
+    ``--include`` and ``--exclude`` expect class names not object IDs
+  * allowed setting the ``OMERO_PASSWORD`` environment variable instead of
+    using the ``-w`` command-line option
+  * made passwords hidden by default when running ``omero config get``
+  * fixed the CLI metadata tablestest plugin to not use an empty list of
+    Columns
+
+This release also upgrades the version of Bio-Formats which OMERO uses to
```
Comment (mtbc) [inline _posts/2017-11-23-omero-5-4-1.md]: Maybe `s/which/that/`? British English is more tolerant of "which" in these cases. Would be glad of a second opinion.

**Comment by jburel** [comment]
Fixes done in https://github.com/openmicroscopy/www.openmicroscopy.org/pull/158

**Pull request closed without merging**